ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml

```diff
@@ -37,7 +37,7 @@ tests:
     - chain: create-infra-move-ingress-monitoring-registry
     workflow: openshift-qe-control-plane
   timeout: 5h0m0s
-- as: ocp-qe-perfscale-aws-cp-24nodes-periodic
+- as: perfscale-aws-control-plane-24nodes-periodic
```

Comment:
I would be ok with keeping this as cp to shorten the name

Reply:
I expanded it to control-plane as some tests already had `control-plane` mentioned and also for any folks outside of the perfscale group it might be more intuitive